di_website/templates/blocks/paragraph_block_footnote.html

```diff
@@ -1,3 +1,3 @@
 {% load string_utils footnotes_utils %}
 
-{% content_with_footnotes self %}
+{% content_with_footnotes self|content %}
```

Comment:
This probably won't make a difference. `content` is imported from string_utils and used in the `content_with_footnotes` function: https://github.com/devinit/DIwebsite-redesign/blob/develop/di_website/footnotes/templatetags/footnotes_utils.py#L4

Reply:
Then I still have no clue why this http://178.128.102.213/blog/priorities-uks-incoming-secretary-state-alok-sharma/ isn't getting stripped ... let me compare the original content